chia/data_layer/data_layer_client.py

```diff
@@ -0,0 +1,117 @@
+import asyncio
+import aiosqlite
+import aiohttp
+import time
+from chia.data_layer.data_store import DataStore
+from chia.util.db_wrapper import DBWrapper
+from chia.types.blockchain_format.tree_hash import bytes32
+from chia.types.blockchain_format.program import Program
+from chia.util.byte_types import hexstr_to_bytes
+from chia.data_layer.data_layer_types import Status, NodeType
+from typing import List, Any, Dict
+
+
+class DataLayerClient:
```

Comment:
what is the purpose of this class? is it a test? It kind of looks like a test except it's not picked up by `pytest`.